.buildkite/scripts/lifecycle/pre_command.sh

```diff
@@ -72,3 +72,8 @@ if [[ "${SKIP_CI_SETUP:-}" != "true" ]]; then
     source .buildkite/scripts/common/setup_bazel.sh
   fi
 fi
+
+PIPELINE_PRE_COMMAND=${PIPELINE_PRE_COMMAND:-".buildkite/scripts/lifecycle/pipelines/$BUILDKITE_PIPELINE_SLUG/pre_command.sh"}
```

Comment:
I ended up not actually using this in this PR, but I wanted to leave it in. You can name pre-command hook in a way that matches the pipeline name, and it will run the hook only for that pipeline. Better than filling this file with `if` statements